server/src/logger.ts

```diff
@@ -72,14 +70,13 @@ function nowString() {
   return `${d.getHours()}:${d.getMinutes()}:${d.getSeconds()}.${d.getMilliseconds()}`;
 }
 
-export class Logger implements ts.server.Logger {
-  private fd = -1;
+class Logger implements ts.server.Logger {
+  private readonly fd: number = -1;
```

Comment:
Should this always be `-1`? Then `loggingEnabled` will never be true.

Reply:
`this.fd` is updated on line 89 if a filename is provided

Reply:
Can we set -1 there instead?

Reply:
done